pkgs/applications/networking/seahub/default.nix

```diff
@@ -59,14 +55,21 @@ python.pkgs.buildPythonApplication rec {
     chardet
     pyjwt
     pycryptodome
+    pyopenssl
+    python-ldap
     qrcode
     pysearpc
-    seaserv
     gunicorn
     markdown
     bleach
+
+    seaserv
   ];
 
+  postPatch = ''
+    sed -i -e "s/SEAFILE_VERSION = '6.3.3'/SEAFILE_VERSION = '${version}'/" seahub/settings.py
```

Comment:
Please use substituteInPlace --replace-fail to notice when this no longer matches.